pkg/shell/shell_test.go

```diff
@@ -100,7 +100,7 @@ func TestWaitForCharacter(t *testing.T) {
 		}
 
 		if strings.TrimSpace(outputString) != expectedOutput {
-			t.Fatalf("Expecting command output to be [%s], got [%s]", expectedOutput, output)
+			t.Fatalf("Expecting command output to be [%s], got [%v]", expectedOutput, output)
```

Comment:
For this one, I'd print the value of `strings.TrimSpace(outputString)` instead of `output`.

```
t.Fatalf("Expecting command output to be [%s], got [%s]", expectedOutput, strings.TrimSpace(outputString))
```